data/en/isstruct.json

```diff
@@ -17,6 +17,22 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+		"title":"isStruct Example",
+		"description":"Returns true if variable is a ColdFusion structure or is a Java object that implements the java.lang.Map interface. ",
+		"code":"someStruct=StructNew(\"Ordered\");\r\nisStruct(someStruct)",
+		"result":true,
+		"runnable":false
```

Comment:
Runnable should be set to true here, because the code example can be run on trycf. You would only set it to false if the example is not a standalone example